parity-crypto/src/publickey/mod.rs

```diff
@@ -44,7 +44,7 @@ use secp256k1::ThirtyTwoByteHash;
 /// converted to a `[u8; 32]` which in turn can be cast to a
 /// `secp256k1::Message` by the `ThirtyTwoByteHash` and satisfy the API for
 /// `recover()`.
-pub struct ZeroesAllowedMessage(H256);
+pub struct ZeroesAllowedMessage(pub H256);
```

Comment:
Forgot this. :/

Reply:
An alternative would be to implement `From<H256> for ZeroesAllowedMessage`, but this is also fine